actions/apply/download.sh

```diff
@@ -11,7 +11,7 @@ onexit() {
 
 DOWNLOAD_FILE=$(mktemp --suffix .tar.gz)
 EXTRACTION_DIR=$(mktemp -d)
-echo "::set-output name=artifacts_dir::${EXTRACTION_DIR}"
+echo "artifacts_dir=${EXTRACTION_DIR}" >> "$GITHUB_OUTPUT"
```

Comment:
nit: I prefer to always use braces on bash variables even when know strictly necessary 
```suggestion
echo "artifacts_dir=${EXTRACTION_DIR}" >> "${GITHUB_OUTPUT}"
```